### PR TITLE
CLI: Add HTML components and stories

### PR DESCRIPTION
**Pull request by tooppaaa**
Issue: #12173

## What I did

Create html components and stories.
Not sure this is the perfect way to go but that does the job

## How to test

- E2E

### CODE REVIEW

**Comment by shilman** [comment]
Looks good @tooppaaa ! My only concern is that all of the examples use JS functions with DOM methods and there are no plain "return a string of HTML" examples, which I think is a common case. Do you think this is an issue for first time users? To me it makes `@storybook/html` look more complicated than it actually is. Maybe I'm overthinking.

**Comment by tooppaaa** [comment]
Opinion shared. What do you think of adding a comment ?
That won't impact stories, we keep it consistent for all frameworks, but shows how to do a story differently ?

File: lib/cli/src/frameworks/html/Button.stories.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    primary: { control: 'boolean' },
+    backgroundColor: { control: 'color' },
+    size: {
+      control: { type: 'select', options: ['small', 'medium', 'large'] },
+    },
+    onClick: { action: 'onClick' },
+  },
+};
+
+const Template = (args) => createButton(args);
```
Comment (tooppaaa) [inline lib/cli/src/frameworks/html/Button.stories.js:16]:
```js
const Template = ({label, ...args}) => {
// You can either use a function to create DOM elements or use a plain html string !
//  return `<div>${label}</div>`;
return createButton({label, ...args});
};
```

**Review by gaetanmaisse — APPROVED** [review]
LGTM! One more time, good job @tooppaaa! 👏  

For the HTML string concern, I have no idea if `@storybook/html` is widely used, if not a comment may be enough.

**Review by ndelangen — APPROVED** [review]

**Pull request merged**